js/.htaccess

```
@@ -0,0 +1,16 @@
+# Apache 2.2
+<IfModule !mod_authz_core.c>
+    Order deny,allow
+    Deny from all
+    <Files ~ "(?i)^.*\.(css|js|gif|png|jpg|svg|html|map|eot|ttf|woff)$">
```

```suggestion
    <Files ~ 

- [ ] "(?i)^.*\.(css|jsjson|gif|png|jpg|svg|html|map|eot|ttf|woff)$">

```
There is the bower.json too.